<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-20T07:07:42 IST" name="Default suite" finished-at="2022-10-20T07:08:18 IST" duration-ms="36118">
    <groups>
    </groups>
    <test started-at="2022-10-20T07:07:42 IST" name="Default test" finished-at="2022-10-20T07:08:18 IST" duration-ms="36118">
      <class name="php.crm.Vtiger.project31.campaignTest.CreatecampaignTest">
        <test-method is-config="true" signature="bsconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:42 IST" name="bsconfig" finished-at="2022-10-20T07:07:42 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsconfig -->
        <test-method is-config="true" signature="bcconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:42 IST" name="bcconfig" finished-at="2022-10-20T07:07:45 IST" duration-ms="3481" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bcconfig -->
        <test-method is-config="true" signature="bmconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:45 IST" name="bmconfig" finished-at="2022-10-20T07:07:48 IST" duration-ms="2293" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmconfig -->
        <test-method retried="true" signature="createCampaignAndVerifyCampaignNameTest()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:48 IST" name="createCampaignAndVerifyCampaignNameTest" finished-at="2022-10-20T07:07:52 IST" duration-ms="4414" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at php.crm.Vtiger.project31.campaignTest.CreatecampaignTest.createCampaignAndVerifyCampaignNameTest(CreatecampaignTest.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCampaignAndVerifyCampaignNameTest -->
        <test-method is-config="true" signature="amconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:52 IST" name="amconfig" finished-at="2022-10-20T07:07:53 IST" duration-ms="1008" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amconfig -->
        <test-method is-config="true" signature="bmconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:53 IST" name="bmconfig" finished-at="2022-10-20T07:07:55 IST" duration-ms="1892" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmconfig -->
        <test-method retried="true" signature="createCampaignAndVerifyCampaignNameTest()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:55 IST" name="createCampaignAndVerifyCampaignNameTest" finished-at="2022-10-20T07:07:58 IST" duration-ms="3362" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at php.crm.Vtiger.project31.campaignTest.CreatecampaignTest.createCampaignAndVerifyCampaignNameTest(CreatecampaignTest.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCampaignAndVerifyCampaignNameTest -->
        <test-method is-config="true" signature="amconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:07:58 IST" name="amconfig" finished-at="2022-10-20T07:08:00 IST" duration-ms="1723" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amconfig -->
        <test-method is-config="true" signature="bmconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:00 IST" name="bmconfig" finished-at="2022-10-20T07:08:02 IST" duration-ms="1932" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmconfig -->
        <test-method retried="true" signature="createCampaignAndVerifyCampaignNameTest()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:02 IST" name="createCampaignAndVerifyCampaignNameTest" finished-at="2022-10-20T07:08:08 IST" duration-ms="5975" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at php.crm.Vtiger.project31.campaignTest.CreatecampaignTest.createCampaignAndVerifyCampaignNameTest(CreatecampaignTest.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCampaignAndVerifyCampaignNameTest -->
        <test-method is-config="true" signature="amconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:08 IST" name="amconfig" finished-at="2022-10-20T07:08:09 IST" duration-ms="972" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amconfig -->
        <test-method is-config="true" signature="bmconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:09 IST" name="bmconfig" finished-at="2022-10-20T07:08:11 IST" duration-ms="2073" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmconfig -->
        <test-method signature="createCampaignAndVerifyCampaignNameTest()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:11 IST" name="createCampaignAndVerifyCampaignNameTest" finished-at="2022-10-20T07:08:15 IST" duration-ms="4392" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at php.crm.Vtiger.project31.campaignTest.CreatecampaignTest.createCampaignAndVerifyCampaignNameTest(CreatecampaignTest.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCampaignAndVerifyCampaignNameTest -->
        <test-method is-config="true" signature="amconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:16 IST" name="amconfig" finished-at="2022-10-20T07:08:18 IST" duration-ms="2082" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amconfig -->
        <test-method is-config="true" signature="aconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:18 IST" name="aconfig" finished-at="2022-10-20T07:08:18 IST" duration-ms="297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aconfig -->
        <test-method is-config="true" signature="asconfig()[pri:0, instance:php.crm.Vtiger.project31.campaignTest.CreatecampaignTest@402bba4f]" started-at="2022-10-20T07:08:18 IST" name="asconfig" finished-at="2022-10-20T07:08:18 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- asconfig -->
      </class> <!-- php.crm.Vtiger.project31.campaignTest.CreatecampaignTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
